src/Rector/Contrib/PHPUnit/SpecificMethod/AssertCompareToSpecificMethodRector.php

```diff
@@ -0,0 +1,124 @@
+<?php declare(strict_types=1);
+
+namespace Rector\Rector\Contrib\PHPUnit\SpecificMethod;
+
+use PhpParser\Node;
+use PhpParser\Node\Expr\FuncCall;
+use PhpParser\Node\Expr\MethodCall;
+use PhpParser\Node\Identifier;
+use PhpParser\Node\Scalar\LNumber;
+use PhpParser\Node\Scalar\String_;
+use Rector\NodeAnalyzer\MethodCallAnalyzer;
+use Rector\Rector\AbstractRector;
+
+/**
+ * Before:
+ * - $this->assertSame($value, {function}($anything), 'message');
+ * - $this->assertNotSame($value, {function}($anything), 'message');
+ * - $this->assertEquals($value, {function}($anything), 'message');
+ * - $this->assertNotEquals($value, {function}($anything), 'message');
+ *
+ * After:
+ * - $this->assert{function}($value, $anything, 'message');
+ * - $this->assertNot{function}($value, $anything, 'message');
+ */
```

Comment:
Could you also add at 1 concrete example? It would make easier to associate with real code

Reply:
Resolved on merge! I think you've already done enough work here.